components/es7210/idf_component.yml

```diff
@@ -0,0 +1,6 @@
+version: 1.0.0-alpha
```

Comment:
Could we start with 1.0.0? Or do you expect any changes in foreseeable time?

Reply:
Sure